server/pbenchinacan/etc/rsyslog.d/50-filter-viaq-tags.conf

```diff
@@ -0,0 +1,6 @@
+# A simple string of "tags" that system administrators can use to identify the
+# various roles a system can take on, or categories to consider it under.
+#
+# E.g.
+# set $!tags = "EXAMPLETAG EXAMPLETAGTWO EXAMPLETAGTHREE";
+set $!tags = "rdu2 perf-dept intlab 3A-Z37 pbench pbench-server staging";
```

Comment:
@portante, I'm assuming that these tags are appropriate for Staging, but not for functional test (or Production), is that correct?  What should they be for functional test?

I can have `run-pbench-in-a-can` set them appropriately for functional test deployment.  (I assume that you are using a static setup for Staging/Production....)

Reply:
> I can have `run-pbench-in-a-can` set them appropriately for functional test deployment.

Or, since this file is completely separate from Staging/Production and is copied from a Git checkout, I _could_ just set the tags statically for functional test.

Reply:
Could this be related to [the below](https://github.com/distributed-system-analysis/pbench/pull/3260#discussion_r1111100887)?

Reply:
Per our discussion, we are going to go with what I have (which probably isn't actually fully-functional...), and we'll iterate.

Next time around, we'll want to try pulling the existing `rsyslog` configuration from the host that we're deploying on  (at least for functional testing; for Staging/Production, the files will come from the local deployment directory), and we'll provide our own tags, including a build-id/PR-number/Git-Hash/etc.